index.html

```diff
@@ -168,69 +228,144 @@ <h2>Project Art Printing Data</h2>
-              <li>Ruby on Rails</li>
-              <li>RsPec</li>
-              <li>CapyBara</li>
-              <li>Selenium</li>
-            </ul>
-      </li>
-      <li id="skillset">
-        <h4>Skills<h4>
-            <ul>
-              <li>Codekit</li>
-              <li>Github</li>
-              <li>Codepen</li>
-              <li>Gitlab</li>
-              <li>Terminal</li>
-            </ul>
-      </li>
 
-    </ul>
+      <div class="grid item12 desktop">
+        <h2>Data Dashboard Healthcare</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard</p>
+        <div class="lang">
+          <ul>
+            <li><button>html</button></li>
+            <li><button>bootstrap</button></li>
+            <li><button>Ruby</button></li>
+          </ul>
+        </div>
+        
+      </div>
+
+
+      <div class="grid item13 desktop">
+        <h2 class="web">Website Portfolio</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard</p>
+        <div class="lang">
+          <ul>
+            <li><button>html</button></li>
+            <li><button>bootstrap</button></li>
+            <li><button>Ruby</button></li>
+          </ul>
+        </div>
+        
+      </div>
+
 
+    </div>
   </section>
 
+
+  <div class="bottom-dark">
+    <section class="about">
+      <h3>ABOUT ME</h3>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button class="see-project btn2">Get my Resume</button>
+
+    </section >
+    <hr class="foot">
+
+    <section class="Programming-skills">
+      <ul>
+        <li id="languages">
+          <h4 class="first">Languages</h4>
+              <ul>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+              </ul>
+        </li>
+        <li id="frameworks" >
+          <h4 class="second">Frameworks</h4>
+              <ul>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RsPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+              </ul>
+        </li>
+        <li id="skillset">
+          <h4 class="third">Skills</h4>
+              <ul>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+              </ul>
+        </li>
+      </ul>
+    </section>
+  </div>
+
   <section class="contact">
     <h3 class="bottom-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
```

Comment:
Good job! but the size of this text should be a little bit larger, and the font-weigh must be bold. So please make sure to fix it.

Yours
![image](https://user-images.githubusercontent.com/34779928/206299509-ac8e9984-d64a-4854-b998-305679ea1eb7.png)

Original
![image](https://user-images.githubusercontent.com/34779928/206299578-97c3f065-7c80-49ed-96a0-98c25cd6ec26.png)